pom.xml

```diff
@@ -253,7 +253,6 @@ under the License.
               <goals>
                 <goal>site</goal>
               </goals>
-              <streamLogsOnFailures>true</streamLogsOnFailures>
```

Comment:
Why?

Reply:
It spills the terminals with thousands of lines of debug output. Basically impossible to scroll back to the invoker report and pray  that your terminal buffer is large enough. Moreover, in other components this isn't enabled also.

Reply:
fair enough but the problem is more: why do we have this for m-invoker-p? (as you complain about verbosity :) )
```
<debug>true</debug>
```

Reply:
I can't tell whether this is enabled for other components as well, but the build.log will contain debug output which is very helpful in failure cases. If it is not enabled, you need to enable it temporarily and and run everything again. Just losing time.